### PR TITLE
get rid of streamifying messages

### PR DESCRIPTION
**Pull request by akosyakov**

### CODE REVIEW

File: src/socket/reader.ts [hunk 1]
```diff
@@ -53,7 +53,8 @@ export class WebSocketMessageReader extends AbstractStreamMessageReader {
         if (this.state === 'initial') {
             this.events.splice(0, 0, { message });
         } else if (this.state === 'listening') {
-            super.readMessage(message, this.callback!);
+            const data = JSON.parse(message);
+            this.callback!(data);
```
Comment (AlexTugarev) [inline src/socket/reader.ts:57]: could it happen that `this.callback` is assigned too late if `readMessage` is called via L40?

Comment (akosyakov) [reply src/socket/reader.ts:57]: good catch, I will move assigning `callback` before processing buffered messages

Comment (akosyakov) [reply src/socket/reader.ts:57]: done

**Review by AlexTugarev — APPROVED** [review]
LGTM!

**Pull request merged**